andela/k2-cfh
# Chore/153009245/setup eslint

#### What does this PR do?

Configure ESLint

#### Description of Task to be completed?

- Install ESLint and other dependencies
- Run eslint --init and choose the airbnb popular style guide
- Configure the generated .eslintrc.yml file to include the airbnb style rules

#### How should this be manually tested?

- Clone online repository
- Navigate to the cloned directory
- Add ESLint as extension to your favourite text editor
- Run npm install
- Open the codebase in your favourite text editor to see linting errors
- Run: npm run eslint on your command prompt to test for linting errors

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

- story type: chore
- story id: 153009245
- story title: Configure ESLint

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2017-11-21 at 3 12 00 pm" src="https://user-images.githubusercontent.com/14099353/33083484-c3a42364-cedf-11e7-8154-2c2cb0ee8b1f.png">
